receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -187,6 +187,48 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
     attributes: []
+  k8s.node.runtime.imagefs.available:
+    enabled: true
```

Comment:
New metrics should be added as optional. If they need to be enabled by default, it should be done via a separate process (by adding warnings first) after the metrics are in place. Please https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md for more details

Reply:
I am very sorry for my foolish mistake that requires me to resubmit the pull request.Thank you for reviewing the code:https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26259